.venv/project/ecommerce/api/storage.py

```diff
@@ -14,4 +14,10 @@ def store_image(file) -> str:
     blob.upload_from_filename(tempFileName) 
     blob.make_public()
 
-    return blob.public_url
\ No newline at end of file
+    return blob.public_url
+
+def get_image_url(fileName):
```

Comment:
Why do we need this? The url should be stored in the item document in the database. This seems problematic

Reply:
Helper function that is not being used. Left it there because it might come in handy for testing and such
.